frappe/www/contact.py

```diff
@@ -22,7 +22,7 @@ def get_context(context):
 	return out
 
 
-max_communications_per_hour = 1000
+max_communications_per_hour = 100
```

Comment:
maybe we can use the `@rate_limit` decorator for this?

Reply:
No idea